client/src/ui/App.tsx

```diff
@@ -28,44 +31,50 @@ export default class App extends Component {
     let dayclass = "afternoon";
 
     if (month < 2 || month > 10) {
-      monthclass = "winter"
-    }
-
-    else if (month > 7) {
-      monthclass = "fall"
-    }
-
-    else if (month > 4) {
-      monthclass = "summer"
-    }
-
-    else {
-      monthclass = "spring"
+      monthclass = "winter";
+    } else if (month > 7) {
+      monthclass = "fall";
+    } else if (month > 4) {
+      monthclass = "summer";
+    } else {
+      monthclass = "spring";
     }
 
     if (time_of_day < 6 || time_of_day >= sunset_end_times[month]) {
-      dayclass = "night"
-    }
-
-    else if (time_of_day >= sunset_start_times[month]) {
-      dayclass = "sunset"
+      dayclass = "night";
+    } else if (time_of_day >= sunset_start_times[month]) {
+      dayclass = "sunset";
     }
 
     return (
-      <div className={"container-fluid full-height background-common background-gradient_" + dayclass + monthclass}>
-        <div className="row">
-          <img src='/logo.svg' className="img-responsive scale-logo-homepage" alt="CU Reviews Logo" />
+      <div
+        className={
+          "container-fluid full-height background-common background-gradient_" +
+          dayclass +
+          monthclass
+        }
+      >
+        <div className='row'>
+          <ProfileDropdown />
+          <img
+            src='/logo.svg'
+            className='img-responsive scale-logo-homepage'
+            alt='CU Reviews Logo'
+          />
         </div>
-        <div className="row homepage-text-padding">
-          <div className="col-lg-7 col-md-7 col-sm-12 col-xs-12">
-            <div className="row">
-              <p className="homepage-text">Search for Cornell courses, rate past classes, and share feedback</p>
+        <div className='row homepage-text-padding'>
+          <div className='col-lg-7 col-md-7 col-sm-12 col-xs-12'>
+            <div className='row'>
+              <p className='homepage-text'>
```

Comment:
formatting nit: are you using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to format your code because I think these should stay as using double quotes `"` instead of single quotes `'`